present/slideshow.py

```diff
@@ -140,12 +168,9 @@ def play(
         repeat=True,
         allow_int=False,
     ):
-        self.reset = [Slide(self, _reset(self.screen), 7, 0)]
+        self.reset = [Scene(_reset(self.screen))]
 
-        self.slides = [
-            Slide(self, self.get_effects(slide), slide.fg_color, slide.bg_color)
-            for slide in self.slides
-        ]
+        self.slides = [Slide(self, slide) for slide in self.slides]
```

Comment:
The refactoring to this file's `Slide` constructor allows this to be simplified. There's still the awkwardness of Slideshow building its slides by passing its existing slides to the _other_ Slide constructor, but at least `get_effects`/`build_scene` is abstracted a bit.